src/Pktsniffer.java

```diff
@@ -42,8 +42,18 @@ public static void main(String[] args) {
         pktsniffer.runSniffer();
     }
 
+    private static void usageAndExit(boolean isError) {
+        System.err.println("""
```

Comment:
Would you like this to be `System.out` instead?

Reply:
No